packages/react-monaco-editor/scripts/copyTypes.ts

```diff
@@ -15,18 +14,18 @@ export function copyTypes() {
     const pkg = packagesToResolve.shift();
     resolvedPackages.push(pkg);
 
-    const packageMatch = pkg.match(/^(@uifabric\/|@fluentui\/)?([\w-]+)/);
-    const dtsPath = expandSourcePath(`${pkg}/dist/${packageMatch[2]}.d.ts`);
+    const packageMatch = pkg.match(/^@fluentui\/([\w-]+)/);
```

Comment:
this changes the previous behaviour, the capturing group was not required or to have 1 occurrence `?`

is this expected behaviour ?

Reply:
It doesn't change the overall behavior because I also updated the next line to use the first group instead of the second. I'm not sure why the scope was optional before.